src/main/java/org/reactome/release/qa/check/OrcidCrossreferenceCheck.java

```diff
@@ -0,0 +1,95 @@
+package org.reactome.release.qa.check;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.stream.Collectors;
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.reactome.release.qa.annotations.SliceQACheck;
+import org.reactome.release.qa.common.AbstractQACheck;
+import org.reactome.release.qa.common.QAReport;
+
+/**
+ * WeeklyQA check to detect person instances that share the same ORCID id (crossreference).
+ *
+ * TODO Check false negative by creating two people with same ORCID.
+ */
+@SliceQACheck
+public class OrcidCrossreferenceCheck extends AbstractQACheck {
+
+    @Override
+    public QAReport executeQACheck() throws Exception {
+	    QAReport report = new QAReport();
+	    report.setColumnHeaders("CrossReference_DBID",
+                                "CrossReference_DisplayName",
+                                "Identifier",
+                                "Person_DBIDs");
+
+	    // Create a map between ORCID and person list.
+	    Map<GKInstance, List<GKInstance>> orcidToPeople = new HashMap<GKInstance, List<GKInstance>>();
+
+	    // Fetch all Person instances from database where crossReference attribute 'IS NOT NULL'.
+	    // Verify that ORCHID database id is being used.
+	    @SuppressWarnings("unchecked")
+        Collection<GKInstance> peopleWithOrcid = dba.fetchInstanceByAttribute(ReactomeJavaConstants.Person,
+                                                                              ReactomeJavaConstants.crossReference,
+                                                                              "IS NOT NULL",
+                                                                              null);
+	    for (GKInstance person : peopleWithOrcid) {
+	        GKInstance crossReference = (GKInstance) person.getAttributeValue(ReactomeJavaConstants.crossReference);
+	        GKInstance refDatabase = (GKInstance) crossReference.getAttributeValue(ReactomeJavaConstants.referenceDatabase);
+	        if (!refDatabase.getDBID().equals(5334734L))
+	            peopleWithOrcid.remove(person);
```

Comment:
Have you got any ConcurrentModificationException during your test? The code most likely will cause this issue since the remove is inside the for loop. If you cannot see this exception, most likely we don't have other cross-reference linked to Persons yet.